src/test/java/pl/kurs/homework/controller/MasterTrafficLightControllerTest.java

```diff
@@ -56,8 +56,9 @@ private static Stream<Arguments> provideArgumentsForChangeLightToGreen() {
         return Stream.of(
                 Arguments.of(65, 100),
                 Arguments.of(25, 48),
+                Arguments.of(23, 234),
```

Comment:
to jest powtórzenie

Reply:
pozniej to zmienie, przelew na koncie, please approve